packages/owl-bot/src/owl-bot.ts

```diff
@@ -255,4 +161,132 @@ export = (privateKey: string | undefined, app: Probot, db?: Db) => {
       Number(context.payload.installation.id)
     );
   });
+}
+
+export async function handlePullRequestLabeled(
```

Comment:
Not blocking, but it will break a capability of running the bot locally with `npm start` (it needs default export).

Reply:
Does `npm start` just works with the wrapper?